GSA-TTS/FAC
# Upload Page Timeout

# Upload Page Timeout (and Some JS Cleaning?)

Issue: #1050 

## Changes:
1. Implemented the 30s timeout for XLSX upload. If the frontend doesn't receive anything by then, it aborts the request and lets the user know.
2. Some minor cleanup of error reporting. All the `info_box` changes and `console.error()`'s are in one place, at least.
3. Moved the submit button below the `info_box`, so users are better able to see the feedback being presented. 

## How to test:
1. Switch to this branch and run normally. 
2. Navigate to an XLSX upload page, like http://localhost:8000/audit/submission-progress/SACIDHERE. 
3. You can try to find an upload that takes longer than 30 seconds, or you can navigate to `/backend/static/js/globals.js` and change it to a short value, such as 500 for 0.5 seconds. 
4. Upload a .xlsx file, and see a relevant error message both on the page and in the browser console.

## Screenshot:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/931e0584-6038-455d-b494-a83fa024f819)
